Guides/Transactions.md

```diff
@@ -0,0 +1,111 @@
+# Swift Driver Transactions Guide
+
+`MongoSwift` 1.0.0 added support for [transactions](https://docs.mongodb.com/manual/core/transactions/), which allow applications to use multi-statement transactions that guarantee the atomicity of reads and writes to multiple documents (in a single or multiple collections). Applications can use transactions instead of implementing complex and error-prone ACID-compliant logic themselves, simplifying development and allowing developers to focus on the new features that really matter.
+
+**Note**: Transactions only work with MongoDB replica sets (v4.0+) and sharded clusters (v4.2+).
+
```

Comment:
I think this guide could use a little prose section here describing the general process of using transactions with the following key points:
- transactions are started on a session
- that session must be passed to each operation in the transaction
- the transaction must be committed
- the transaction can be aborted

Reply:
Done.